config/components.yml

```diff
@@ -39,6 +39,7 @@ jres:
 # The JavaOpts is last in order to allow any user-defined JAVA_OPTS to override contributions from earlier frameworks
 frameworks:
   - "JavaBuildpack::Framework::MultiBuildpack"
+  - "JavaBuildpack::Framework::TakipiAgent"
```

Comment:
Can you please elaborate on why this is necessary?

Reply:
If the Takipi agent is loaded after the APM agent, it will take longer to start the app on PCF/PWS. Many times it will reach the timeout limit and not be able to deploy/start the app.

Reply:
Doesn't this work in reverse as well though?  Since APM agents are instrumenting a bunch of stuff doesn't that all slow down if OverOps is in place already?  There's a fundamental "slow startup when adding an agent" problem that isn't OverOps-specific that you deal with [by setting `CF_STARTUP_TIMEOUT`](https://docs.cloudfoundry.org/devguide/deploy-apps/large-app-deploy.html#consid_limits).